humanoid_league_team_communication/scripts/show_team_comm.py

```diff
@@ -32,16 +34,14 @@
         SIDE: UNDEFINED
 """
 
-
-class TeamCommPrinter:
-
+class TeamCommPrinter(Node):
```

Comment:
Inheriting from `Node` here was easier. Is that ok or should I switch to a composition approach?

Reply:
It's fine